### ISSUE TITLE
Error while reading data into buffer: Passphrase required for this entry

### ISSUE THREAD

**Issue by zyphs21** [sunflsks/LAWrapper#1]
Hello, Thanks for your work. When I use extractArchive function for an zip file with passphrase, it shows the error: 
```
Error while reading data into buffer: Passphrase required for this entry
```
Any idea about how to extract file with passphrase ?  Maybe create a function which has a password parameter in it?

**Comment by sunflsks** [comment]
Should be fixed by 3a46446e227a0fa8859da23f3eff857ee8aa1da2; let me know if it works out or if you still have issues.

**Comment by zyphs21** [comment]
Thanks！It works out
